packages/create-styles/src/cssCustomProperties/createRootStore.js

```diff
@@ -0,0 +1,97 @@
+import { getPropValue } from './getPropValue';
+import { hasVariable, isCustomProperty } from './utils';
+
+/**
+ * Stores CSS config variables that are expected to be added to :root.
+ * This is used for CSS variable fallback handling for IE 11.
+ */
```

Comment:
Type correction: `state` is probably better typed as `Record<string, string>` rather than a plain `object` type. That should be corrected throughout and you can alias it using `@typedef {Record<string, string>} State` if you'd like (probably for the best).

Reply:
@saramarcondes I apologize... I don't know how to do this 🙈.